template-test/ci-run.sh

```diff
@@ -11,9 +11,14 @@ VARIANT="${VARIANT:-basic}"
 APP_NAME="${VARIANT}-test-app"
 
 rm -rf template-test/dummy/$APP_NAME
-cd template-test/dummy
+mkdir -p template-test/dummy/$APP_NAME
 
-echo -e $GENERATOR_INPUT | RACK_ENV=development RAILS_ENV=development rails new $APP_NAME -d postgresql -m $TEMPLATE
+# otherwise rails webpacker:install initiates yarn init and that overrides these yarn.lock package.json files 
+# exist under the root directory instead of creating new ones under template-test/dummy/$APP_NAME
+( cd template-test/dummy/$APP_NAME && yarn init --yes )
```

Comment:
I just realised we actually should be doing this in the template 🤦

Reply:
This script expects the cwd to be `template-test/dummy` for the commands run after yarn init too.

Reply:
Cool, this is done moved into frontend-base template. It worked this time with the --yes params